File: Aplicacion Movil/MedMonitor/app/src/main/java/Models/AppointmentDTO.kt
package Models

import java.io.Serializable

data class AppointmentDTO(
    val id: Long? = null,
    val usuario_id: Long,
    val fecha: String, // En formato "YYYY-MM-DD"
    val hora: String, // En formato "HH:MM"
    val descripcion: String
) : Serializable

File: Aplicacion Windows/TFG_Windows/TFG/BD/AppointmentDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TFG.BD
{
    public class AppointmentDTO
    {
        public long id { get; set; }

        public long usuario_id { get; set; }

        public string fecha { get; set; } // formato "yyyy-MM-dd"

        public string hora { get; set; } // formato "HH:mm"

        public string descripcion { get; set; }
    }
}
